internal/repository/copy.go

```diff
@@ -68,11 +68,7 @@ func (r *Copy) CopyFile(
 	if err != nil {
 		return err
 	}
-	defer func() {
-		if e := out.Close(); e != nil {
-			err = e
-		}
-	}()
+	defer func() { _ = out.Close() }()
```

Comment:
See #150 for my rationale here.  One way or another, I **will** see this destroyed 😆

Reply:
That makes good sense